### ISSUE TITLE
[request] dark mode for the whole site

### ISSUE THREAD

**Issue by Just-some-one** [java/devjava-content#106]
i think it would be nice to have dark mode plus the current light mode 
steering on the screen for long time is not comfortable in the light mode

**Comment by nasrob** [comment]
I agree that it would be a great addition to our current light mode. I'd like to work on this feature. However, I have a couple of questions:

- Is there a specific theme or style guide we should follow for the dark mode design?
- Are there any particular elements or sections of the app that you think would benefit most from a dark mode treatment?

**Comment by danthe1st** [comment]
> I'd like to work on this feature.

Note that the version on the website has some things (like a search) that aren't available in this repository so I don't know whether you could really contribute it as you couldn't properly test it with whatever they changed in the UI since then.

**Comment by carimura** [comment]
Great idea. i don't think much has changed in terms of CSS. I could try and sync the two repos but this might not be necessary to get a dark mode theme working. As for theme or style guide, we do our best to follow the WCAG rules, one in particular for dark theme might be contrast ratio of 7 to 1 [1] [2].

[1] https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=141%2C146#contrast-enhanced

[2] https://www.w3.org/WAI/WCAG22/Techniques/general/G17.html
